scripts/rspec/rspec-templates/TestMethod.VB.cs

```diff
@@ -0,0 +1,6 @@
+﻿        // FIXME: Move this up
+        private readonly VerifierBuilder builderVB = new VerifierBuilder<$DiagnosticClassName$>();
```

Comment:
Why didn't you add the logic directly to move this up where it belongs?
I think it can be done easily just like the part where we insert the VbNet using.
Just find the line with the builderCS and insert it in the next line? WDYT?

Reply:
We don't use the VB part so often that it would make sense. Finding the line is quite hard because we don't know what's the suffix of the previous line is. I might improve it later if it starts bothering me :)

Reply:
I moved the `//FIXME` the the EOL to make it easier to move, but I see I didn't push that change.

Reply:
Cant you find the `private readonly VerifierBuilder` substring of the CS one and just insert the VBNet part before it with a line break in between. That should do the job and I would say it is not super complicated to do.

Reply:
Suggestion number 2 :) Find the first semicolon after `private readonly VerifierBuilder builderCS = new VerifierBuilder` and insert it there?

Reply:
That would not work nicely with scenarios like this
https://github.com/SonarSource/sonar-dotnet/blob/f34bf4f014ae7faff27f19c8846b9f1034aaf042/analyzers/tests/SonarAnalyzer.UnitTest/Rules/DeclareTypesInNamespacesTest.cs#L29-L30

In general, it's too much for something not that valuable. I'll see if I'll need to improve it later, because it's going to be me who adds the most VB rules anyway.